cmd/telemetry.go

```diff
@@ -27,6 +29,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const gaugeTelemetryEnabled = "GAUGE_TELEMETRY_ENABLED"
+
+var telemetryEnv = os.Getenv(gaugeTelemetryEnabled)
+
+func telemetryEnabled() bool {
+	if telemetryEnv == "" {
+		return config.TelemetryEnabled()
+	}
```

Comment:
if `telemetryEnv` is empty then `strconv.ParseBool` will fail and return `config.TelemetryEnabled()` which is what the extra check is returning. It won't even reach to Line 44.